templates/deploy_fortigate_autoscale.hybrid_licensing.json

```
@@ -1001,6 +1018,13 @@
                     "StorageAccountName": {
                         "value": "[variables('storageAccountName')]"
                     },
+                    "SyncRecoveryCount": {
+                        "value": "[parameters('SyncRecoveryCount')]"
+                    },
+                    "TerminateUnhealthyVM": {
+                        "value": "[variables('terminateUnhealthyVM')]"
```

pass the value of the parameter to the link template 
```suggestion
                        "value": "[parameters('TerminateUnhealthyVM')]"
```